main.py

```diff
@@ -17,7 +21,8 @@ def instruction():
           "\n - Number can be a float number (6.66)")
 
 
-def validate_number(num):
+def validate_number(num: any) -> bool:
```

Comment:
you don't have to use 'any' here. you're validating if passed string is a number.
so:
(num: str)
is enough here :)